### PR TITLE
[FIX] fieldervice_account_payment

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Review by bodedra — APPROVED** [review]
LGTM :+1:

**Review by nelsonramirezs — APPROVED** [review]
LGTM

**Review by nelsonramirezs — APPROVED** [review]

**Review by brian10048 — APPROVED** [review]

File: fieldservice_account_payment/models/account_payment.py [hunk 1]
```diff
@@ -38,4 +38,5 @@ def _compute_fsm_order_ids(self):
         fsm_order_ids = []
         for invoice in self.invoice_ids:
             fsm_order_ids.append(invoice.fsm_order_ids.id)
-        self.fsm_order_ids = [(6, 0, fsm_order_ids)]
+        if len(fsm_order_ids) > 0:
```
Comment (dreispt) [inline fieldservice_account_payment/models/account_payment.py]:
```suggestion
        if fsm_order_ids:
```

**Review by dreispt — APPROVED** [review]

**Pull request merged**